<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoryConfig</class>
 <widget class="QWidget" name="DirectoryConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Directories</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Videos path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="m_VideosPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_SelectVideosPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Temporary path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="m_TempPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_SelectTempPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Alerts path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="m_AlertsPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_SelectAlertsPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Backup path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="m_BackupPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_SelectBackupPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_SelectTempPath</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>selectTempPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_TempPath</sender>
   <signal>editingFinished()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>onTempPathChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_SelectVideosPath</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>selectVideoPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_VideosPath</sender>
   <signal>editingFinished()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>onVideoPathChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_AlertsPath</sender>
   <signal>editingFinished()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>onAlertsPathChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_SelectAlertsPath</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>selectAlertsPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_BackupPath</sender>
   <signal>editingFinished()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>onBackupPathChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_SelectBackupPath</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryConfig</receiver>
   <slot>selectBackupPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectVideoPath()</slot>
  <slot>onVideoPathChanged()</slot>
  <slot>selectTempPath()</slot>
  <slot>onTempPathChanged()</slot>
  <slot>selectAlertsPath()</slot>
  <slot>onAlertsPathChanged()</slot>
  <slot>selectBackupPath()</slot>
  <slot>onBackupPathChanged()</slot>
 </slots>
</ui>
